R/persistence-class.R

```diff
@@ -25,6 +25,8 @@
 #' @param ... Parameters passed to methods.
 #' @param dimension A non-negative integer specifying the homology dimension for
 #'   which to recover a matrix of persistence pairs.
+#' @param birth Numeric; the height at which to declare all leaves were born.
```

Comment:
To be consistent with the way I have been documenting input parameters, I would rephrase into *A numeric value specifying the height at which to declare all leaves were born.*